Subject: do not redeclare standard time functions on musl

The old K&R declarations conflict with the prototypes supplied by musl's
time.h.

--- a/unix/unxcfg.h
+++ b/unix/unxcfg.h
@@ -117,7 +117,6 @@
 #else
 #  include <time.h>
-   struct tm *gmtime(), *localtime();
 #endif
 
 #if (defined(BSD4_4) || (defined(SYSV) && defined(MODERN)))
